arduino/smartcar/smartcar.ino

```diff
@@ -37,10 +41,12 @@ void loop()
 }
 
 void detectObstacle() {
- const auto distance = front.getDistance();
+  const auto frontDistance = front.getDistance();
+  const auto backDistance = back.getDistance();
+
   // When distance is `0` it means there's no obstacle detected
   //When car is within an obstacle range of 0-1.5 meters, it stops
-  if (distance > 0 && distance < 150) {
+  if ((frontDistance > 0 && frontDistance < 150) || (backDistance > 0 && backDistance < 150)) {
```

Comment:
At a later point we should discuss the distance we chose here, but for now it looks good!